.github/workflows/contribution-check.yml

```
@@ -5,14 +5,13 @@ on:
     types: [opened, edited, synchronize]
 
 jobs:
-  commit-message-check:
+  lint-pr:
```

I suggest something which more explicit.
```suggestion
  pr-title:
```